app/views/includes/scripts.njk

```diff
@@ -40,6 +40,10 @@
     window.GOVUKFrontend.initAll();
     if (mainWrap.classList.contains('charge-new')) {
       showCardType().init();
+      window.payScripts.helpers.setGlobalChargeId();
```

Comment:
As discussed it looks like the `showCardType()` scrits end up using this global chargeId - it would be good for this to be as high up as possible to make sure everything has access.

Reply:
Updated..